website/docs/reference/rbac.md

```diff
@@ -9,7 +9,7 @@ This document forms the specifications for [Role-Based Access Control](https://e
 
 Unleash has two levels in its hierarchy of resources:
 
-1. **Global resources** - Everything that lives across the entire Unleash instance. Examples of this include:
+1. **Root resources** - Everything that lives across the entire Unleash instance. Examples of this include:
```

Comment:
Do we want to switch from Global resources to root resources? I think switching from Global Roles to Root Roles makes sense, root resources feel a bit weird IMO. But, maybe it is just me

Reply:
Yeah, I had the exact same thought, but I think I did the same thing you did